test/vstest.console.UnitTests/Internal/ConsoleLoggerTests.cs

```diff
@@ -563,15 +564,75 @@ public void TestResultHandlerShouldWriteToConsoleShouldShowPassedTestsForNormalV
+            loggerEvents.EnableEvents();
+            var parameters = new Dictionary<string, string>
+            {
+                { "verbosity", "quiet" },
+                { DefaultLoggerParameterNames.TargetFramework , ".NETFramework,version=v4.5.1"}
+            };
+            this.consoleLogger.Initialize(loggerEvents, parameters);
+
+            foreach (var testResult in this.GetTestResultsObject())
+            {
+                loggerEvents.RaiseTestResult(new TestResultEventArgs(testResult));
+            }
+
+            foreach (var testResult in this.GetPassedTestResultsObject())
+            {
+                loggerEvents.RaiseTestResult(new TestResultEventArgs(testResult));
+            }
+
+            loggerEvents.RaiseTestRunComplete(new TestRunCompleteEventArgs(new Mock<ITestRunStatistics>().Object, false, false, null, new Collection<AttachmentSet>(), TimeSpan.FromSeconds(1)));
+            loggerEvents.WaitForEventCompletion();
+
+            this.mockOutput.Verify(o => o.WriteLine(string.Format(CultureInfo.CurrentCulture, CommandLineResources.TestRunSummaryPassed, 2, 1, 0, 1, "1 m 2 s", "TestSourcePassed", "net451"), OutputLevel.Information), Times.Once);
+            this.mockOutput.Verify(o => o.WriteLine(string.Format(CultureInfo.CurrentCulture, CommandLineResources.TestRunSummaryFailed, 5, 1, 1, 1, "1 h 6 m", "TestSource", "net451"), OutputLevel.Information), Times.Once);
+        }
+
+        [TestMethod]
+        [DataRow("normal")]
+        [DataRow("detailed")]
+        public void TestResultHandlerShouldNotShowformattedFailedTestsAndPassedTestsForOtherThanQuietVebosity(string verbosityLevel)
+        {
+            var loggerEvents = new InternalTestLoggerEvents(TestSessionMessageLogger.Instance);
+            loggerEvents.EnableEvents();
+            var parameters = new Dictionary<string, string>
+            {
+                { "verbosity", verbosityLevel },
+                { DefaultLoggerParameterNames.TargetFramework , "net451"}
+            };
+            this.consoleLogger.Initialize(loggerEvents, parameters);
+
+            foreach (var testResult in this.GetTestResultsObject())
+            {
+                loggerEvents.RaiseTestResult(new TestResultEventArgs(testResult));
+            }
+
+            foreach (var testResult in this.GetPassedTestResultsObject())
+            {
+                loggerEvents.RaiseTestResult(new TestResultEventArgs(testResult));
+            }
+
+            loggerEvents.RaiseTestRunComplete(new TestRunCompleteEventArgs(new Mock<ITestRunStatistics>().Object, false, false, null, new Collection<AttachmentSet>(), TimeSpan.FromSeconds(1)));
+            loggerEvents.WaitForEventCompletion();
+
+            this.mockOutput.Verify(o => o.WriteLine(string.Format(CultureInfo.CurrentCulture, CommandLineResources.TestRunSummaryPassed, 2, 1, 0, 1, "1 m 2 s", "TestSourcePassed", "net451"), OutputLevel.Information), Times.Never);
```

Comment:
Can we have a verify on mockOutput.Information that includes the console color as parameter?

Reply:
No, Ioutput doesnot contain informatin member.